nft/Cargo.toml

```diff
@@ -9,15 +9,15 @@ edition = "2018"
 
 [dependencies]
 codec = { package = "parity-scale-codec", version = "1.3.0", default-features = false }
-sp-std = { version = "2.0.0", default-features = false }
-sp-runtime = { version = "2.0.0", default-features = false }
+sp-std = {  version = "2.0.0", default-features = false, git = "https://github.com/paritytech/substrate", rev = "b2750359" }
```

Comment:
please revert chose change and update Cargo.dev.toml instead.

I've already bumped Substrate version at #350 so no need to bump again